uncertainty_toolbox/recalibration.py

```diff
@@ -146,3 +146,44 @@ def obj(ratio):
             opt_ratio = 1.0
 
     return opt_ratio
+
+
+def get_std_recalibrator(y_mean, y_std, y_true, criterion="ma_cal"):
+    std_recal_ratio = optimize_recalibration_ratio(
+        y_mean, y_std, y_true, criterion
+    )
+
+    def std_recalibrator(std_arr):
+        return std_recal_ratio * std_arr
+
+    return std_recalibrator
+
+
+def get_quantile_recalibrator(y_pred, y_std, y_true):
+    exp_props, obs_props = uct.get_proportion_lists_vectorized(
+        y_pred, y_std, y_true, prop_type='quantile'
+    )
+    iso_model = iso_recal(exp_props, obs_props)
+
+    def quantile_recalibrator(y_pred, y_std, quantile):
+        recal_quantile = uct.metrics_calibration.get_quantile(
+            y_pred, y_std, quantile, recal_model=iso_model
+        )
+        return recal_quantile
+
+    return quantile_recalibrator
+
+
+def get_pi_recalibrator(y_pred, y_std, y_true):
```

Comment:
ok potential idea: would it be more clear to call this the `interval_calibrator` and this function `get_interval_calibrator`? It wasn't immediately clear to me what `pi` referred, to, and also we use the string `interval` for prop_type, so we'd have consistency with this.

Reply:
(i realize this would require a fair bit of changes throughout if we decided to make this change)